cns/restserver/restserver.go

```diff
@@ -119,6 +121,12 @@ type Response struct {
 	Message    string
 }
 
+// GetEndpointResponse describes response from the The GetEndpoint API.
+type GetEndpointResponse struct {
+	Response     Response     `json:"response"`
+	EndpointInfo EndpointInfo `json:"endpointInfo"`
+}
+
```

Comment:
I think this type is redundant, the response to a GET on `/network/endpoints/<endpoint>` should just be the `<endpoint>`. Leverage headers and response codes!